src/component/admin/groups/Group/Group.tsx

```diff
@@ -133,28 +134,34 @@ export const Group: VFC = () => {
                 Cell: ({ row: { original: rowUser } }: any) => (
                     <ActionCell>
                         <Tooltip title="Edit user" arrow describeChild>
-                            <IconButton
-                                onClick={() => {
-                                    setSelectedUser(rowUser);
-                                    setEditUserOpen(true);
-                                }}
-                            >
-                                <Edit />
-                            </IconButton>
+                            <span>
+                                <IconButton
+                                    disabled={group?.users.length == 1}
```

Comment:
Small thing but `==` in javascript allows type coercion. `===` will do a strict check on value and type. It's a good pattern to default to `===`

Reply:
Fixed